apps/site/src/lib/admin/useParticipants.ts

```diff
@@ -38,12 +38,20 @@ function useParticipants() {
 		console.log("Checking in", participant);
 		// TODO: implement mutation for showing checked in on each day
 		await axios.post(`/api/admin/checkin/${participant._id}`);
+		mutate(data);
```

Comment:
Issue: I'm pretty sure if you provide a data object, it needs to be the updated data. We can do a full mutation by not providing anything, see `submitReview` in `useApplicant`.